parm/config/gfs/config.aero

```diff
@@ -20,6 +20,9 @@ case ${machine} in
   "WCOSS2")
     AERO_INPUTS_DIR="/lfs/h2/emc/global/noscrub/emc.global/data/gocart_emissions"
     ;;
+  "GAEA")
+    AERO_INPUTS_DIR="/gpfs/f5/epic/proj-shared/global/glopara/data/ICSDIR/gocart_emissions"
```

Comment:
gocart_emissions should not be filed under `ICSDIR`.  `ICSDIR` is the initial conditions directory